app/components/build-layout.js

```diff
@@ -21,5 +25,8 @@ export default Component.extend({
       jobs.forEach((j) => j.get('config'));
       return jobs.isEvery('isLoaded') && jobs.isEvery('isConfigLoaded') && stagesAreLoaded;
     }
-  )
+  ),
+
+  isConfig: match('router.currentRouteName', /config$/),
```

Comment:
`:oreally!?!:` 

i didn't know! i'll try that out.

Reply:
ah, i now get it. yeah, we've added a `reads` property for that ... changed.

Reply:
Yep, since it's established above as a computed property, you should be able to use it in another CP. It doesn't super make a difference in this case, but since it is there and a little bit shorter to type, why not :)